src/NetEscapades.EnumGenerators/NetEscapades.EnumGenerators.csproj

```diff
@@ -13,11 +13,12 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.CodeAnalysis.Analyzers" Version="3.3.2" PrivateAssets="all" />
     <PackageReference Include="Microsoft.CodeAnalysis.CSharp" Version="4.0.1" PrivateAssets="all" />
+    <PackageReference Include="System.ComponentModel.Annotations" Version="5.0.0" Condition="'$(TargetFramework)' == 'net48'" />
```

Comment:
I don't think we should add this to the Source generator project itself? 🤔 I think we can probably get away with only adding it to the test projects - that's what users would need to do to use the attribute I believe? 

```suggestion
```

Reply:
You're right. Fixed.